src/qibo/optimizers.py

```diff
@@ -1,5 +1,6 @@
 def optimize(loss, initial_parameters, args=(), method='Powell',
-             options=None, compile=False, processes=None):
+             options=None, compile=False, processes=None, jac=None, hess=None,
+             hessp=None, bounds=None, constraints=(), tol=None, callback=None):
```

Comment:
Do you think it would be useful to add the `args` argument here to allow user to pass additional arguments to the loss? I think it is not useful for existing Qibo models but could be if the user defines a custom variational model.

Also, shall we make the order of arguments consistent with scipy by moving the `options` flag in the end?

Reply:
> Do you think it would be useful to add the args argument here...

We have already this flag here, right (3rd argument)?

> Also, shall we make the order of arguments consistent with scipy by moving the options flag in the end?

yes, I agree.

Reply:
> > Do you think it would be useful to add the args argument here...
> 
> We have already this flag here, right (3rd argument)?

Indeed, sorry I was looking at the `minimize` method arguments in `models.py` when writing this comment where we don't have the `args` argument but there it is probably not needed anyway. No need to change anything here.